ajax/libs/cssobj/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "cssobj",
+  "description": "Runtime CSS manager, Turn CSS into dynamic JS module, Stylesheet CRUD (Create, Read, Update, Delete) in CSSOM, Solve common problems of CSS-in-JS, name space (local) class names",
+  "filename": "cssobj.min.js",
+  "keywords": [
+    "cssinjs",
+    "css-in-js",
+    "css in js",
+    "cssom",
+    "auto-prefixer",
+    "css generate",
+    "diff css",
+    "css modules"
+  ],
+  "author": "James Yang <jamesyang999@gmail.com>",
+  "homepage": "https://cssobj.github.io/cssobj-demo/",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cssobj/cssobj.git"
+  },
+  "npmName": "cssobj",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/cssobj.!(amd|cjs|es)*"
```

Comment:
@futurist Please explain here all types of files that you have in you dist folder, to avoid further questions